xbmc/cores/paplayer/AudioDecoder.cpp

```diff
@@ -58,7 +58,10 @@ void CAudioDecoder::Destroy()
   m_pcmBuffer.Destroy();
 
   if ( m_codec )
+  {
+    m_codec->DeInit();
```

Comment:
It change in behaviour nearly nothing, the only new thing is that a function who was not called on audio decoder add-on's before is called now.

Not sure if it was never called or it was lost on a update of PAPlayer.